setup-nightfall

```diff
@@ -1,4 +1,25 @@
 #! /bin/bash
+
+usage()
+{
+  echo "Usage:"
+  echo "  -g or --ganache; for a ganache simulator"
+}
+
+if [ ! -z "$1" ]; then
+  while [ -n "$1" ]; do
```

Comment:
Why do we need to force this from the setup phase? We did never require it. What happens if we want to deploy to a different blockchain? If you need to just accelerate test, then create a new setup file with you changes, but do not modify this one